test/NotificationActivationTests/pch.cpp

```diff
@@ -0,0 +1,4 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
```

Comment:
Use new copyright notice:
// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.